weed/shell/command_volume_fsck.go

```diff
@@ -126,20 +139,22 @@ func (c *commandVolumeFsck) Do(args []string, commandEnv *CommandEnv, writer io.
 		return fmt.Errorf("read filer buckets path: %v", err)
 	}
 
-	collectMtime := time.Now().UnixNano()
+	c.collectStartTime = time.Now()
```

Comment:
should be cutoffTime.

"start time" means starting from some time.

Reply:
ok